main.cpp

```diff
@@ -0,0 +1,11 @@
+#include <hwlib.hpp>
```

Comment:
There shouldn't be a main.cpp located in this folder.

Reply:
You are correct, I have removed it.